Error when building on Debian

``` Dockerfile
FROM debian:11


RUN mkdir -p /home/pi
WORKDIR /home/pi
RUN apt update && apt upgrade
RUN cd /home/pi && mkdir projects


RUN apt install -y mesa-vulkan-drivers vulkan-tools libvulkan-dev git cmake clang clinfo

# build and install OpenCL
RUN apt install -y mesa-vulkan-drivers vulkan-tools libvulkan-dev cmake clang clinfo python3 pocl-opencl-icd libhwloc-dev ocl-icd-opencl-dev
RUN cd projects && \
    git clone https://github.com/kpet/clvk.git && cd clvk && \
    git submodule update --init --recursive && \
    python3 ./external/clspv/utils/fetch_sources.py --deps llvm
RUN apt install -y build-essential
RUN cd projects/clvk && mkdir -p build && cd build && cmake -DARM_TARGET=8 -DLLVM_INCLUDE_BENCHMARKS=OFF -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF -DLLVM_ENABLE_BINDINGS=OFF -DLLVM_ENABLE_UNWIND_TABLES=OFF -DLLVM_BUILD_TOOLS=OFF -DCLSPV_BUILD_SPIRV_DIS=OFF -DCLSPV_BUILD_TESTS=OFF -DCLVK_BUILD_TESTS=OFF -DCLVK_BUILD_SPIRV_TOOLS=OFF -DCLVK_ENABLE_SPIRV_IL=OFF ../
RUN cd projects/clvk/build && make -j$(nproc)
```

I have Dockerfile and trying to build clvk on debian, but some error appears.

```

#0 8088.7 /home/pi/projects/clvk/src/log.cpp:392:10: error: 'VK_VENDOR_ID_POCL' was not declared in this scope; did you mean 'VK_VENDOR_ID_MESA'?
#0 8088.7   392 |     case VK_VENDOR_ID_POCL:
#0 8088.7       |          ^~~~~~~~~~~~~~~~~
#0 8088.7       |          VK_VENDOR_ID_MESA
#0 8093.8 make[2]: *** [src/CMakeFiles/OpenCL-objects.dir/build.make:173: src/CMakeFiles/OpenCL-objects.dir/log.cpp.o] Error 1
#0 8093.8 make[2]: *** Waiting for unfinished jobs....
#0 8113.1 make[1]: *** [CMakeFiles/Makefile2:36309: src/CMakeFiles/OpenCL-objects.dir/all] Error 2
#0 8113.3 make: *** [Makefile:171: all] Error 2

```

--- a/src/log.cpp
+++ b/src/log.cpp
@@ -396,8 +396,10 @@ std::string vulkan_vendor_id_string(VkVendorId vid) {
         return "Codeplay Software Ltd.";
     case VK_VENDOR_ID_MESA:
         return "Mesa";
+#ifdef VK_VENDOR_ID_POCL
     case VK_VENDOR_ID_POCL:
         return "PoCL";
+#endif
 #ifdef VK_VENDOR_ID_MOBILEYE
     case VK_VENDOR_ID_MOBILEYE:
         return "Mobileye";

